README.md

```diff
@@ -1,11 +1,53 @@
-# *A*ndroid *L*ibrary for *P*assport *I*nformation via *N*FC *E*xtraction
+# **A**ndroid **L**ibrary for **P**assport **I**nformation via **N**FC **E**xtraction
 
-## Usage
+
+ALPINE is a library to simplify the extraction of passport information by reading data off of the NFC chip
+
+## Download
 
 Clone the repository
 
 `git clone git@github.com:nimblehq/alpine.git`
 
+## Usage
+
+### Scanning Passport MRZ
+
+MRZ reading is performed by the `MrzProcessor` by passing in a filepath and a callback to receive the result.
+
+To create an instance of `MrzProcessor`, call `newInstance()`
+
+```kotlin
+val mrzProcessor = MrzProcessor.newInstance()
+```
+
+To extract the MRZ from the passport, call `processImageFile()`
+
+```kotlin
+val imageFilePath: String = imageFile.absolutePath
+val mrzProcessorResultListener = object : MrzProcessorResultListener {
+    override fun onSuccess(mrzInfo: MrzInfo) {}
+    override fun onError(e: MrzProcessorException) {}
+}
+mrzProcessor.processImageFile(imageFilePath, mrzProcessResultL)
```

Comment:
@ryan-conway Should it be `mrzProcessorResultListener` instead of `mrzProcessResultL` 🤔 
```suggestion
mrzProcessor.processImageFile(imageFilePath, mrzProcessorResultListener)
```